src/Microsoft.Azure.WebJobs.Host/Loggers/UpdateParameterLogCommand.cs

```diff
@@ -18,11 +19,12 @@ internal sealed class UpdateParameterLogCommand : IRecurrentCommand
         private readonly IReadOnlyDictionary<string, IWatcher> _watches;
         private readonly IStorageBlockBlob _parameterLogBlob;
         private readonly TraceWriter _trace;
+        private readonly ILogger _logger;
 
         private string _lastContent;
 
         public UpdateParameterLogCommand(IReadOnlyDictionary<string, IWatcher> watches,
-            IStorageBlockBlob parameterLogBlob, TraceWriter trace)
+            IStorageBlockBlob parameterLogBlob, TraceWriter trace, ILogger logger)
```

Comment:
That's the plan eventually. I did it this way so it would be fairly easy to rip out TraceWriter and have ILogger live in its place. That change will have to happen in a major release since it's a big break. I believe we wanted to do it when we move everything to .NET Core.